vulnfeeds/cmd/pypi/main.go

```diff
@@ -140,12 +163,18 @@ func main() {
 			v, notes := vulns.FromCVE(id, cve.CVE)
 			v.AddPkgInfo(pkgInfo)
 			versions, versionNotes := cves.ExtractVersionInfo(cve.CVE, validVersions)
+
 			notes = append(notes, versionNotes...)
 			v.Affected[0].AttachExtractedVersionInfo(versions)
 			if len(v.Affected[0].Ranges) == 0 {
 				log.Printf("No affected versions detected.")
 			}
 
+			if *excludeUnbounded && anyUnbounded(v) {
+				log.Printf("Skipping %s as we could not find a fixed version.", cve.CVE.ID)
```

Comment:
Nit: adjust wording to state it's unbounded, since `last_affected` is also checked for